packages/mobile/src/screens/collection-screen/CollectionScreenDetailsTile.tsx

```diff
@@ -170,6 +174,16 @@ export const CollectionScreenDetailsTile = ({
 
   const isReachable = useSelector(getIsReachable)
 
+  const { data: currentUserId } = useGetCurrentUserId({})
+  const { data: collection } = useGetPlaylistById(
+    {
+      playlistId: typeof collectionId === 'number' ? collectionId : -1,
```

Comment:
I'm not quite following, whats the scenario for this?

Reply:
`CollectionId` could be a `SmartCollectionVariant` in which case we don't want to do anything. Ideally these cases would be split out into different components.

<img width="454" alt="Screenshot 2024-05-16 at 3 50 09 PM" src="https://github.com/AudiusProject/audius-protocol/assets/3893871/b83d65a7-26b6-43d2-8d10-b2b99a2e819f">

Reply:
* Would be nice to include a comment here explaining why the special number
* Worth mentioning we _could_ update the useGetPlaylistById() hook to support the special IDs and return something compatible here (if that makes sense)
* Agreed that it should be separate code paths for the two types. Taking a "collectionId" that isn't actually an `ID` is code smell.